lib/App.js

```diff
@@ -65,12 +64,17 @@ App.prototype._init = function() {
   this._waitForAttach = true;
   this._cancelAttach = false;
   this.model = new this.derby.Model();
+  var serializedViews = this._views();
   serializedViews(derbyTemplates, this.views);
   // Must init async so that app.on('model') listeners can be added.
   // Must also wait for content ready so that bundle is fully downloaded.
   this._contentReady();
 };
 
+App.prototype._views = function () {
+  return require('derby/lib/_views');
```

Comment:
Without testing it, I'm unsure whether this'll work, since we're already inside the `derby` library. Safer to keep it as `require('./_views')`, IMO.